src/test/java/com/epam/jamp2/model/CalculationCommandTest.java

```diff
@@ -0,0 +1,39 @@
+package com.epam.jamp2.model;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+import java.math.BigDecimal;
+
+import static org.hamcrest.CoreMatchers.*;
+import static org.junit.Assert.*;
+
+/**
+ * Created by Northzzn on 2017/2/27.
+ */
+@RunWith(JUnit4.class)
+public class CalculationCommandTest {
+
+    @Test
+    public void testParseFromStringWithAdd() throws CommandFormatException {
+        CalculationCommand calculationCommand = CalculationCommand.parseFromString("CNY10.0+USD20.01=CFH");
+
+        assertThat(calculationCommand.getLeftOperand().getCurrencyCode().get(), is(equalTo("CNY")));
```

Comment:
Minor. It's better to define constants for currency codes and use that constants in your tests instead of hardcoded values.